Microsoft.Toolkit.Services/Services/MicrosoftTranslator/TranslatorService.cs

```diff
@@ -138,22 +128,11 @@ public async Task<string> DetectLanguageAsync(string text)
         /// <para>For more information, go to https://docs.microsofttranslator.com/text-translate.html#!/default/get_GetLanguagesForTranslate.
         /// </para>
         /// </remarks>
+        /// <seealso cref="GetLanguageNamesAsync(string)"/>
         public async Task<IEnumerable<string>> GetLanguagesAsync()
```

Comment:
Inherit should also work here on the docs

Reply:
Done